src/layout/Footer/Footer.svelte

```diff
@@ -45,7 +45,7 @@
 				{@html Discord}
 			</IconButton>
 		</div>
-		<p>Copyright © 2021, The Files Authors</p>
+		<p>Copyright © {new Date().getFullYear()}, The Files Authors</p>
```

Comment:
The year indicates the year since when the copyright claim is active (usually the year it was first published in). Since we don't want to only claim copyright on the latest version, we should either keep 2021 or write "2021 - 2022".

Reply:
Should be fixed now.

Reply:
Is this being done on the server side or could this be changed by setting the date on my machine to e.g. 1970?

Reply:
It is dependent on the client year.

Reply:
I don't think we need to account for the very rare edge case of someone setting their computer to a different year than the actual year, though.

Reply:
It seems that this section has not much legal meaning (at least in the US) and having 2021 be in the section would be fine here.